install_script_opw.sh

```diff
@@ -0,0 +1,644 @@
+#!/bin/bash
+# (C) Datadog, Inc. 2010-present
+# All rights reserved
+# Licensed under Apache-2.0 License (see LICENSE)
+# Datadog Observability Pipelines Worker installation script:
+# install and set up the Observability Pipelines Worker on supported Linux distributions
+# using the package manager and Datadog repositories.
+
+set -e
+
+install_script_version="0.1.0"
+logfile="dd-install.log"
+support_email="support@datadoghq.com"
+
+# DATADOG_APT_KEY_CURRENT.public always contains key used to sign current
+# repodata and newly released packages
+# DATADOG_APT_KEY_382E94DE.public expires in 2022
+# DATADOG_APT_KEY_F14F620E.public expires in 2032
+# DATADOG_APT_KEY_C0962C7D.public expires in 2028
+APT_GPG_KEYS=("DATADOG_APT_KEY_CURRENT.public" "DATADOG_APT_KEY_C0962C7D.public" "DATADOG_APT_KEY_F14F620E.public" "DATADOG_APT_KEY_382E94DE.public")
+
+# DATADOG_RPM_KEY_CURRENT.public always contains key used to sign current
+# repodata and newly released packages
+# DATADOG_RPM_KEY_E09422B3.public expires in 2022
+# DATADOG_RPM_KEY_FD4BF915.public expires in 2024
+# DATADOG_RPM_KEY_B01082D3.public expires in 2028
+RPM_GPG_KEYS=("DATADOG_RPM_KEY_CURRENT.public" "DATADOG_RPM_KEY_B01082D3.public" "DATADOG_RPM_KEY_FD4BF915.public" "DATADOG_RPM_KEY_E09422B3.public")
```

Comment:
Same for `DATADOG_RPM_KEY_E09422B3.public` as above.

Reply:
Resolved in [5bda0aa](https://github.com/DataDog/agent-linux-install-script/pull/63/commits/5bda0aa8b2b5d7b4c3262460d7012aff26b1a4c5)